### PR TITLE
pango: update to 1.48.9.

### PR DESCRIPTION
**Pull request by ardadem**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [X] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by biopsin** [comment]
Seems changelog url is moved, so far I only found
https://gitlab.gnome.org/GNOME/pango/-/raw/main/NEWS

**Comment by ardadem** [comment]
`changelog="https://gitlab.gnome.org/GNOME/pango/-/raw/main/NEWS"`

It's the same url?

**Comment by biopsin** [comment]
Doh, I must have loaded the url from the old template.. sorry for the fuzz

File: srcpkgs/pango/template [hunk 1]
```diff
@@ -1,20 +1,21 @@
 # Template file for 'pango'
 pkgname=pango
-version=1.48.7
+version=1.48.9
 revision=1
 build_style=meson
 build_helper=gir
-configure_args="-Dintrospection=$(vopt_if gir enabled disabled)"
+configure_args="-Dintrospection=$(vopt_if gir enabled disabled)
+ --wrap-mode nofallback"
```
Comment (ericonr) [inline srcpkgs/pango/template:8]: Why did you have to add this? Might make sense as an option in the build style.

Comment (ardadem) [reply srcpkgs/pango/template:8]: Oh, i added to skip fallback on gi docgen locally, should have been remove it before push.

**Pull request closed without merging**